### PR TITLE
Add versionadded directives to ssl.minimum_version and ssl.maximum_version

### PR DESCRIPTION
**Pull request by zmwangx**
`ssl.minimum_version` and `ssl.maximum_version` were introduced in Python 3.7, alongside `ssl.TLSVersion` and such. Not having `versionadded` directives is somewhat confusing.

This is trivial enough and warrants skip issue and skip news.

### CODE REVIEW

**Review by csabella — APPROVED** [review]

**Review by vstinner — DISMISSED** [review]
I would prefer to see "versionadded" after the note, not before. See other docs like bz2, decimal, gzip, http.client, etc. for other examples. I prefer to be consistent for the documentation.

**Comment by csabella** [comment]
@vstinner, thanks for the review.  I had checked that and the ssl page seems to have both, mostly the changes for 3.7 and 3.8 appear before the note and older ones appear after the note.  Do you think the others on this `ssl.rst` should be cleaned up to be consistent?

**Comment by vstinner** [comment]
As soon as ssl doc is inconsistent for note before/after version added, @csabella you can merge the PR if you want. This minor inconsistent shouldn't hold this PR.

> @vstinner, thanks for the review. I had checked that and the ssl page seems to have both, mostly the changes for 3.7 and 3.8 appear before the note and older ones appear after the note.

Oh sorry, I didn't notice that.

I consider that the note has more value than the "version added", and so should be read first.

> Do you think the others on this `ssl.rst` should be cleaned up to be consistent?

I don't know. Is there a common style for Python documentation somewhere, cc @JulienPalard ?

**Comment by zmwangx** [comment]
I fixed the order throughout `ssl.rst` (`versionadded` after `note`).

**Comment by zmwangx** [comment]
I have made the requested changes; please review again.

**Comment by csabella** [comment]
Looking through other doc pages, it does appear that most of them have the versionadded after the note (as you originally pointed out) and since adjusting it here was minimal, I think the PR looks good.  Do you agree @vstinner?  Thanks!

**Comment by vstinner** [comment]
@tiran: Since you likely added many of these "versionadded", would you mind to review the PR? The PR just moves "versionadded" after notes. IMHO notes are more important than versionadded, and so should be read first.

**Review by vstinner — APPROVED** [review]
LGTM, but I would prefer to see @tiran review first.

@tiran: FYI @csabella (new core dev) wants to be the one who merges this PR ;-)

**Review by tiran — APPROVED** [review]
LGTM. thanks!

**Comment by csabella** [comment]
Thank you for the PR, @zmwangx!

**Pull request merged**
